spec/core_helpers.rb

```diff
@@ -22,13 +22,9 @@ def core_cache
 
   def populate_core(cache)
     terms = [
+      ['personauthorities', 'person', '2020', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(20201599147149106)'2020'"],
       ['personauthorities', 'person', 'Kev', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(Kev1599058769862)'Kev'"],
-    ]
-    populate(cache, terms)
-  end
-
-  def populate_core(cache)
-    terms = [
+      ['personauthorities', 'person', 'Mark Smith', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(MarkSmith)'Mark Smith'"],
```

Comment:
Some terms hadn't been added to the cache, so weren't getting transformed to refnames